tests/pip_test_requirements.txt

```diff
@@ -1,4 +1,5 @@
 coverage~=7.2.2
+pre-commit~=3.7.0
```

Comment:
Do we include this package here instead of the main service pip requirements because it's test related, even though it applies to the entire repository?

Reply:
Yup - exactly that (or test-ish). I guess a better name might be `dev-requirements` or something. But it's not something the service code needs to run when people make requests, so we don't want it in there.